app/index.html

```diff
@@ -90,7 +94,7 @@
                         <a class="page-scroll" href="explore.html">Explore</a>
                     </li>
                     <li>
-                        <a class="page-scroll" href="login.html" onclick = "initalizeFirebase();firebase.auth().signOut()">Logout</a>
+                        <a class="page-scroll" href="login.html" onclick = "firebase.auth().signOut()">Logout</a>
```

Comment:
It is because I call initalizeFirebase() in index.js. If I don't delete it, login.html can't be reached by click the logout button as the firebase is initalized twice.